src/dos/program_autotype.cpp

```diff
@@ -0,0 +1,158 @@
+#include <algorithm>
```

Comment:
(just a note for now)

Thanks for skipping copyright for now. We need to discuss how we want to handle copyright assignment for new files, that were not inherited from upstream. This is a separate topic, probably best handled via email until we'll agree on the solution.

Reply:
Oh, good catch (I had actually forgot to add it!); will coordinate over email.
Leaving this open until decision made.